test/apps/iodemo/io_demo.cc

```diff
@@ -1354,6 +1361,12 @@ static int parse_args(int argc, char **argv, options_t *test_opts)
         case 'q':
             test_opts->validate = true;
             break;
+        case 'H':
+            UcxLog::use_human_time = true;
```

Comment:
maybe make `test_opts` static and visible everywhere in io_demo.cc? can be wrapped as classic singleton.
it would simplify passing global parameters in many places

Reply:
good idea, can it be in another PR?

Reply:
sure